draft-ietf-mls-protocol.md

```diff
@@ -1904,48 +1905,44 @@ tree with the same set of nodes and edges as the epoch's ratchet tree. Each
 leaf in the AS Tree is associated with the same group member as the
 corresponding leaf in the ratchet tree. Nodes are also assigned an index
 according to their position in the array representation of the tree (described
-in {{tree-math}}). If V is a node in the AS Tree then IndexOf(V) denotes it's
-index while V.leftChild and V.rightChild denote the children of V (if they
-exist).
+in {{tree-math}}). If V is a node index in the AS Tree then left(V)
```

Comment:
`V` is just horrible, can you just call it `n`,`ni` or `i` for node, node index or index instead ?